Fixed PR1 issues #382

tdegeorge opened the pull request:
Fixed #382

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -526,7 +532,14 @@
               <li> <p> <m>(-1,-4)</m>; minimum   </p> </li>
               <li> <p> <m>x=-1</m>   </p> </li>
-              <li> <p> (placeholder for graph)   </p> </li>
+              <li> <p> <figure width="100%" xml:id="graph-of-x^2+2*x-3">
```
```suggestion
              <li> <p> <figure width="100%" xml:id="graph-of-x2+2x-3">
```
I don't think ^ or * are allowed in xml:ids, which was causing the build to fail.

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -542,7 +555,14 @@
               <li> <p> <m>(3,20)</m>; maximum   </p> </li>
               <li> <p> <m>x=3</m>   </p> </li>
-              <li> <p> (placeholder for graph)   </p> </li>
+              <li> <p> <figure width="100%" xml:id="graph-of--5(x-3)^2+20">
```
```suggestion
              <li> <p> <figure width="100%" xml:id="graph-of--5(x-3)2+20">
```

tdegeorge replied:
Should I use a different name? I initially had like "graph-of-part-a" but thought that was too vague

siwelwerd replied:
I don't mind this kind of name since it is descriptive, but there are strict rules about what characters are allowed in the xml:id.  The parenthesis and + may not be allowed either...

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -542,7 +555,14 @@
               <li> <p> <m>x=3</m>   </p> </li>
-              <li> <p> (placeholder for graph)   </p> </li>
+              <li> <p> <figure width="100%" xml:id="graph-of--5(x-3)^2+20">
+        <image xml:id="graph-of--5(x-3)^2+20">
```
Need a distinct xml:id again

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -558,7 +578,14 @@
                 <li> <p> <m>(-3,-5)</m>; minimum   </p> </li>
                 <li> <p> <m>x=-3</m>   </p> </li>
-                <li> <p> (placeholder for graph)   </p> </li>
+                <li> <p> <figure width="100%" xml:id="graph-of-5x^2+30x+40">
```
```suggestion
                <li> <p> <figure width="100%" xml:id="graph-of-5x2+30x+40">
```

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -558,7 +578,14 @@
                 <li> <p> <m>x=-3</m>   </p> </li>
-                <li> <p> (placeholder for graph)   </p> </li>
+                <li> <p> <figure width="100%" xml:id="graph-of-5x^2+30x+40">
+        <image xml:id="graph-of-5x^2+30x+40">
```
Need a distinct xml:id again

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -574,14 +601,20 @@
                   <li> <p> <m>(-4,-3)</m>; minimum   </p> </li>
                   <li> <p> <m>x=-4</m>   </p> </li>
-                  <li> <p> (placeholder for graph)   </p> </li>
+                  <li> <p> <figure width="100%" xml:id="graph-of-2(x+4)^2-3">
```
```suggestion
                  <li> <p> <figure width="100%" xml:id="graph-of-2(x+4)2-3">
```

tdegeorge commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -526,7 +532,14 @@
               <li> <p> <m>x=-1</m>   </p> </li>
-              <li> <p> (placeholder for graph)   </p> </li>
+              <li> <p> <figure width="100%" xml:id="graph-of-x^2+2*x-3">
+        <image xml:id="graph-of-x^2+2*x-3">
```
That's weird because it showed up in the preview

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -574,14 +601,20 @@
                   <li> <p> <m>x=-4</m>   </p> </li>
-                  <li> <p> (placeholder for graph)   </p> </li>
+                  <li> <p> <figure width="100%" xml:id="graph-of-2(x+4)^2-3">
+        <image xml:id="graph-of-2(x+4)^2-3">
```
Need a distinct xml:id again

StevenClontz commented:
https://github.com/TeamBasedInquiryLearning/library/actions/runs/12383373632/job/34565926169#step:6:20

👆 `graph-of-x2+2x-3` is not a legal xml:id

StevenClontz commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -526,7 +532,14 @@
               <li> <p> <m>(-1,-4)</m>; minimum   </p> </li>
               <li> <p> <m>x=-1</m>   </p> </li>
-              <li> <p> (placeholder for graph)   </p> </li>
+              <li> <p> <figure width="100%" xml:id="graph-of-x2+2x-3">
+        <image xml:id="graph-of-x^2+2*x-3">
```
```suggestion
              <li> <p> <figure width="100%" xml:id="PR1-fluency-graph-1-figure">
        <image xml:id="PR1-fluency-graph-1-image">
```

StevenClontz commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -542,7 +555,14 @@
               <li> <p> <m>(3,20)</m>; maximum   </p> </li>
               <li> <p> <m>x=3</m>   </p> </li>
-              <li> <p> (placeholder for graph)   </p> </li>
+              <li> <p> <figure width="100%" xml:id="graph-of--5(x-3)2+20">
+        <image xml:id="graph-of--5(x-3)^2+20">
```
```suggestion
              <li> <p> <figure width="100%" xml:id="PR1-fluency-graph-2-figure">
	        <image xml:id="PR1-fluency-graph-2-image">
```

siwelwerd commented:
> 👆 `graph-of-x2+2x-3` is not a legal xml:id

Wait, why not?  It's alphanumeric and dashes

StevenClontz commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -558,7 +578,14 @@
                 <li> <p> <m>(-3,-5)</m>; minimum   </p> </li>
                 <li> <p> <m>x=-3</m>   </p> </li>
-                <li> <p> (placeholder for graph)   </p> </li>
+                <li> <p> <figure width="100%" xml:id="graph-of-5x2+30x+40">
+        <image xml:id="graph-of-5x^2+30x+40">
```
etc.

StevenClontz commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -574,14 +601,20 @@
                   <li> <p> <m>(-4,-3)</m>; minimum   </p> </li>
                   <li> <p> <m>x=-4</m>   </p> </li>
-                  <li> <p> (placeholder for graph)   </p> </li>
+                  <li> <p> <figure width="100%" xml:id="graph-of-2(x+4)2-3">
+        <image xml:id="graph-of-2(x+4)^2-3">
```
etc.

tdegeorge commented:
Please check to see if I submitted correctly...:)

I also added in another activity that I saw from issue #381  - I figured I would do it at the same time....

AbbyANoble commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -12,8 +12,9 @@
     
-    <observation>Quadratic functions have many different applications in the real world. For example, say we want to identify a point at which the maximum profit or minimum cost occurs. Before we can interpret some of these situations, however, we will first need to understand how to read the graphs of quadratic functions to locate these least and greatest values. 
-       
+    <observation><p>Quadratic functions have many different applications in the real world. For example, say we want to identify a point 
+    at which the maximum profit or minimum cost occurs. Before we can interpret some of these situations, however, we will first need to 
+    understand how to read the graphs of quadratic functions to locate these least and greatest values. </p>
```
```suggestion
    <observation><p>Quadratic functions have many different applications in the real world. For example, say we want to identify a point at which the maximum profit or minimum cost occurs. Before we can interpret some of these situations, however, we will first need to understand how to read the graphs of quadratic functions to locate these least and greatest values. </p>
```

AbbyANoble replied:
removing paragraph breaks

AbbyANoble commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -200,6 +201,87 @@
+      </p>
+    </answer>
+  </task>
+
+  <task>
+    <statement>
+      <p>
+        Factor the perfect square trinomial. 
+      </p>
+    </statement>
+    <answer>
+      <p>
+        <me>
+          y+1=(x-2)^2
+        </me>
+      </p>
+    </answer>
+  </task>
+
+  <task>
+    <statement>
+      <p>
+        Now isolate <m>y</m> and then determine the vertex and axis of symmetry.
+      </p>
+    </statement>
+    <answer>
+      <p>
+        <m>y=(x-2)^2-1</m>
+      </p>
+      <p>
+        Axis of Symmetry: <m>x=2</m>
+      </p>
+      <p>
+        Vertex: <m>(2,-1)</m>
+      </p>
+    </answer>
+  </task>
+    
+  </activity>
+
```
See comment in #381!

AbbyANoble replied:
Pasting it here so it's easier:

> I added an activity to have students convert from standard to vertex. Does that work?

I mentioned this mostly to build the idea in general. To develop the fact that the x value of the vertex is -b/2a by completing the square on y=ax^2+bx+c, not a specific quadratic. Currently, the text just says x=-b/2a without any reasoning behind it, so I thought completing the square to get to vertex form would show that -b/2a pop up where we expect the h in vertex form.

As is, it seems like this activity is encouraging students that the way to find the vertex and aos of a standard form quadratic is by completing the square, which is not we'd actually want students to do.

I mentioned this could be in an appendix because maybe it's too in the weeds for a lot of people since there is a lot of algebraic manipulation of the a, b, and c. Or if it's in the text, maybe show the whole thing, and then ask what the x-value of the vertex is?

tdegeorge replied:
Oh ok, I misinterpreted that then. I vote that if we do include it (with the general quadratic), we put it in the appendix. When teaching this course, I don't normally have students work on the general case.

tdegeorge replied:
@kathypinzon @siwelwerd 

Do you guys have a preference? I also think it's nice to have students do a specific example to show that the two forms are related. While we don't normally want students to find the vertex and such in this way, it's nice to see how to move from one form to another.

siwelwerd replied:
>As is, it seems like this activity is encouraging students that the way to find the vertex and aos of a standard form quadratic is by completing the square, which is not we'd actually want students to do.

I think if you asked me to find the vertex and axis of symmetry of a quadratic, I'd complete the square... what are you wanting your students to do?

tdegeorge replied:
I think most people would either have students find the vertex and AOS from vertex form or tell them to graph it and find it on the graph.

siwelwerd replied:
But how do you get vertex form?  Isn't that the completing the square part?

AbbyANoble replied:
> > As is, it seems like this activity is encouraging students that the way to find the vertex and aos of a standard form quadratic is by completing the square, which is not we'd actually want students to do.
> 
> I think if you asked me to find the vertex and axis of symmetry of a quadratic, I'd complete the square... what are you wanting your students to do?

I would use x=-b/2a over completing the square pretty much every time. And that's what I show my students too as a method if they had standard form. The idea I had in the suggestion was to show where -b/2a comes from (which is completing the square on the standard form to get to vertex form). I would not expect a student to complete the square to find the vertex, though it would be fine with me if they did.

AbbyANoble replied:
> I think most people would either have students find the vertex and AOS from vertex form or tell them to graph it and find it on the graph.

I would tell them to use -b/2a! I do show them the completing the square bit, but I don't have them do it because it's messy and frustrating and not something I'd ask them to do again in a general sense.

AbbyANoble replied:
> @kathypinzon @siwelwerd
> 
> Do you guys have a preference? I also think it's nice to have students do a specific example to show that the two forms are related. While we don't normally want students to find the vertex and such in this way, it's nice to see how to move from one form to another.

I'm fine with keeping the specific example in there, but I think it needs an additional question at the end that connects it to -b/2a to motivate the observation that the vertex is (-b/2a,f(-b/2a)). And I think it would be easier to see that with an a other than 1. Otherwise is just looks like the vertex is -b/2.

AbbyANoble commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -526,7 +614,14 @@
               <li> <p> <m>(-1,-4)</m>; minimum   </p> </li>
               <li> <p> <m>x=-1</m>   </p> </li>
-              <li> <p> (placeholder for graph)   </p> </li>
+              <li> <p> <figure width="100%" xml:id="PR1-fluency-graph-1-figure">
+        <image xml:id="PR1-fluency-graph-1-image">
+          <sageplot>
+            f(x) = (x^2+2*x-3)
+              p=plot(f, (x, -6, 6), ymin=-5, ymax=20, color='blue', thickness=3)
+          </sageplot>
+        </image>
+      </figure>   </p> </li>
```
```suggestion
              <li> <p> <figure width="100%" xml:id="PR1-fluency-graph-1-figure">
        <image xml:id="PR1-fluency-graph-1-image">
          <sageplot>
            f(x) = (x^2+2*x-3)
              p=plot(f, (x, -6, 6), ymin=-5, ymax=20, color='blue', thickness=3)
              p+=point([(-1,-4),(-2,f(-2)),(-3,f(-3)),(0,f(0)),(1,f(1))],pointsize=50,color='blue')
          </sageplot>
        </image>
      </figure>   </p> </li>
```

AbbyANoble replied:
Adds in vertex and additional points.

AbbyANoble commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -542,7 +637,14 @@
               <li> <p> <m>(3,20)</m>; maximum   </p> </li>
               <li> <p> <m>x=3</m>   </p> </li>
-              <li> <p> (placeholder for graph)   </p> </li>
+              <li> <p> <figure width="100%" xml:id="PR1-fluency-graph-2-figure">
+        <image xml:id="PR1-fluency-graph-2-image">
+          <sageplot>
+            f(x) = (-5*(x-3)^2+20)
+              p=plot(f, (x, -1, 7), ymin=-5, ymax=30, color='blue', thickness=3)
+          </sageplot>
+        </image>
+      </figure>   </p> </li>
```
```suggestion
              <li> <p> <figure width="100%" xml:id="PR1-fluency-graph-2-figure">
        <image xml:id="PR1-fluency-graph-2-image">
          <sageplot>
            f(x) = (-5*(x-3)^2+20)
              p=plot(f, (x, -1, 7), ymin=-5, ymax=30, color='blue', thickness=3)
              p+=point([(3,f(3)),(2,f(2)),(1,f(1)),(4,f(4)),(5,f(5))],pointsize=50,color='blue')
          </sageplot>
        </image>
      </figure>   </p> </li>
```

AbbyANoble commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -558,7 +660,14 @@
                 <li> <p> <m>(-3,-5)</m>; minimum   </p> </li>
                 <li> <p> <m>x=-3</m>   </p> </li>
-                <li> <p> (placeholder for graph)   </p> </li>
+                <li> <p> <figure width="100%" xml:id="PR1-fluency-graph-3-figure">
+        <image xml:id="PR1-fluency-graph-3-image">
+          <sageplot>
+            f(x) = (5*x^2+30*x+40)
+              p=plot(f, (x, -6, 2), ymin=-10, ymax=30, color='blue', thickness=3)
+          </sageplot>
+        </image>
+      </figure>  </p> </li>
```
```suggestion
                <li> <p> <figure width="100%" xml:id="PR1-fluency-graph-3-figure">
        <image xml:id="PR1-fluency-graph-3-image">
          <sageplot>
            f(x) = (5*x^2+30*x+40)
              p=plot(f, (x, -6, 2), ymin=-10, ymax=30, color='blue', thickness=3)
              p+=point([(-3,f(-3)),(-4,f(-4)),(-5,f(-5)),(-2,f(-2)),(-1,f(-1))],pointsize=50,color='blue')
          </sageplot>
        </image>
      </figure>  </p> </li>
```

AbbyANoble commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -574,14 +683,20 @@
                   <li> <p> <m>(-4,-3)</m>; minimum   </p> </li>
                   <li> <p> <m>x=-4</m>   </p> </li>
-                  <li> <p> (placeholder for graph)   </p> </li>
+                  <li> <p> <figure width="100%" xml:id="PR1-fluency-graph-4-figure">
+        <image xml:id="PR1-fluency-graph-4-image">
+          <sageplot>
+            f(x) = (2*(x+4)^2-3)
+              p=plot(f, (x, -8, 4), ymin=-5, ymax=30, color='blue', thickness=3)
+          </sageplot>
+        </image> 
+      </figure>  </p> </li>
```
```suggestion
                  <li> <p> <figure width="100%" xml:id="PR1-fluency-graph-4-figure">
        <image xml:id="PR1-fluency-graph-4-image">
          <sageplot>
            f(x) = (2*(x+4)^2-3)
              p=plot(f, (x, -8, 4), ymin=-5, ymax=30, color='blue', thickness=3)
              p+=point([(-4,f(-4)),(-5,f(-5)),(-6,f(-6)),(-3,f(-3)),(-2,f(-2))],pointsize=50,color='blue')
          </sageplot>
        </image> 
      </figure>  </p> </li>
```

tdegeorge commented:
@siwelwerd @AbbyANoble Please check this. I changed the activity and added in a remark, but the remark might be too long? I wasn't sure how to show where the -b/2a comes from without actually doing it....

StevenClontz commented:
> Discussion today: still need to add remark or something between 4.1.10 and 4.1.11 saying "If you do this in general... the − b 2 a pops out". Perhaps change previous activity to have a be different from one.

Is the precal group meeting via Zoom regularly Tuesdays this spring? Share a calendar invite to my email and I'm happy to pop in when possible.

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -200,6 +199,121 @@
+  <task>
+    <statement>
+      <p>
+        Notice that the <m>a</m>-value is not <m>1</m>. Before we complete the square, we will want to factor out the GCF from the <m>x^2</m> and <m>x</m> terms. What does your equation look like when factoring out the GCF?
```
Don't we want to factor out the coefficient of $x^2$, not the greatest common factor?

tdegeorge replied:
Ahh! You're right - no idea why I said that!

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -200,6 +199,121 @@
+    
+    <me>y-c=a\left(x^2+\frac{b}{a}x\right)</me>
+    
+    <me>y-c+a\left(\frac{b}{2a}\right)^2=a\left(x^2+\frac{b}{a}x+\frac{b}{2a}\right)^2</me> 
```
```suggestion
    <me>y-c+a\left(\frac{b}{2a}\right)^2=a\left(x^2+\frac{b}{a}x+\left(\frac{b}{2a}\right)^2\right)</me> 
```

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -200,6 +199,121 @@
+    
+    <me>y-c+a\left(\frac{b}{2a}\right)^2=a\left(x^2+\frac{b}{a}x+\frac{b}{2a}\right)^2</me> 
+    
+    <me>y-c+a\left(\frac{b}{2a}\right)^2=a\left(x^2+\frac{b}{2a}\right)^2</me>
```
```suggestion
    <me>y-c+a\left(\frac{b}{2a}\right)^2=a\left(x+\frac{b}{2a}\right)^2</me>
```

tdegeorge commented:
Ok.....Getting this message: 
<img width="536" alt="Screenshot 2025-01-23 at 9 06 52 AM" src="https://github.com/user-attachments/assets/5c553a67-09a9-4e6e-a1c0-d33aae215854" />

tdegeorge commented:
Are you able to see my changes? I think I addressed all the comments above

tdegeorge commented:
It doesn't look like it synced my changes to Observation 4.1.12....

siwelwerd commented:
My guess is you committed my suggestions in the browser, but then didn't fetch those changes to your codespace before you made other changes

tdegeorge commented:
How do I fix that? Is there a way to go back?

siwelwerd commented:
@StevenClontz What's the best fix for Tonya?

AbbyANoble commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -200,9 +199,124 @@
+  </activity>
+
+  <remark>
+    <p>In <xref ref="standard-to-vertex-form"/>, we were able to convert from standard form to vertex form. If you were to do this for the general form of <m>y=ax^2+bx+c</m>, you will see that the axis of symmetry will always be <m>-\frac{b}{2a}</m>. For instance, suppose we start with
```
```suggestion
    <p>In <xref ref="standard-to-vertex-form"/>, we were able to convert from standard form to vertex form. If you were to do this for the general form of <m>y=ax^2+bx+c</m>, you will see that the <m>x</m>-value of the vertex will always be of the form <m>-\frac{b}{2a}</m>. For instance, suppose we start with
```

I think this should have been about the x-value? Since the idea was to find the vertex?

tdegeorge replied:
I made the changes in the new branch because I apparently keep messing up on this one! :-p

AbbyANoble commented:
A lot of graphs weren't showing up for me in the preview. Assuming a preview issue and not a graph issue? Not really sure.

StevenClontz commented:
> @StevenClontz What's the best fix for Tonya?

Create a new branch, then make a pull request from the new branch (either into main or into this branch, whatever's more appropriate.

tdegeorge commented:
> A lot of graphs weren't showing up for me in the preview. Assuming a preview issue and not a graph issue? Not really sure.

Do you mean the codechat? I was able to see the graphs so I am hoping all is okay.....

tdegeorge commented:
Ok, made a separate branch and submitted a pull request.....

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -200,9 +199,124 @@
+    
+    <me>y-c+a\left(\frac{b}{2a}\right)^2=a\left(x+\frac{b}{2a}\right)^2</me>
+    
+    <me>y=a\left(x^2+\frac{b}{2a}\right)^2+c-\frac{b^2}{4a}</me>
```
```suggestion
    <me>y=a\left(x+\frac{b}{2a}\right)^2+c-\frac{b^2}{4a}</me>
```

One more stray exponent to fix.

siwelwerd commented on source/precalculus/source/04-PR/01.ptx:
```diff
@@ -200,9 +199,124 @@
   <observation xml:id="standard-form-aos">
     <statement>
-      <p> Just as with the vertex form of a quadratic, we can use the standard form of a quadratic to find the <term>axis of symmetry</term> <idx><h>quadratic function</h><h>axis of symmetry</h></idx> and the <term>vertex</term> <idx><h>quadratic function</h><h>vertex from standard form</h></idx>by using the values of <m>a, b </m>, and <m> c </m>. Given the standard form of a quadratic, the axis  ...
+      <p> Just as with the vertex form of a quadratic, we can use the standard form of a quadratic<m>\left(f(x)=ax^2+bx+c\right)</m> to find the <term>axis of symmetry</term> <idx><h>quadratic function</h><h>axis of symmetry</h></idx> and the <term>vertex</term> <idx><h>quadratic function</h><h>vertex from standard form</h></idx>by using the values of <m>a, b </m>, and <m> c </m>. Given the stand ...
```
```suggestion
      <p> Just as with the vertex form of a quadratic, we can use the standard form of a quadratic <m>\left(f(x)=ax^2+bx+c\right)</m> to find the <term>axis of symmetry</term> <idx><h>quadratic function</h><h>axis of symmetry</h></idx> and the <term>vertex</term> <idx><h>quadratic function</h><h>vertex from standard form</h></idx>by using the values of <m>a, b </m>, and <m> c </m>. Given the standard form of a quadratic, the axis of symmetry is the vertical line <m> x=-\dfrac {b}{2a}</m> and the vertex is at the point <m> \left(-\dfrac{b}{2a},f\left(-\dfrac{b}{2a}\right)\right)</m>.
```

Missing space

siwelwerd commented:
I think this is good to go... @AbbyANoble maybe you can give it one more look and then approve and merge it.

AbbyANoble commented:
> > A lot of graphs weren't showing up for me in the preview. Assuming a preview issue and not a graph issue? Not really sure.
> 
> Do you mean the codechat? I was able to see the graphs so I am hoping all is okay.....

I mean in the previews that are generated and linked above. Not sure why I can't see the images. I guess I need to fire up a codespace.

<img width="286" alt="Screenshot 2025-02-02 at 10 38 48 AM" src="https://github.com/user-attachments/assets/cc7a68fe-7d8e-4822-ad05-90a5bddef58e" />

siwelwerd commented:
> I mean in the previews that are generated and linked above. Not sure why I can't see the images. I guess I need to fire up a codespace.

Yes, this is a design choice so that the previews build quickly.  Use a codespace to check images.

StevenClontz commented:
Well, kind of. It should be building images for sections that have been edited, but it seems that those in answer "knowls" are not building...

AbbyANoble commented:
Well I was going to fire up a codespace but I don't know how to look at something there that hasn't been merged or isn't something I'm personally editing and previewing.

siwelwerd commented:
Just change it to this branch (Tonya-PR-Issue)

siwelwerd commented:
@AbbyANoble I am seeing the four graphs in the browser preview now.
